p2panda-rs/src/schema/error.rs

```diff
@@ -5,6 +5,48 @@ use thiserror::Error;
 
 use crate::schema::SchemaId;
 
+/// Custom errors related to `SchemaName`.
+#[derive(Clone, Error, Debug)]
+pub enum SchemaNameError {
+    /// Encountered a malformed schema name.
+    #[error("Schema name contains too many or invalid characters")]
+    MalformedSchemaName,
+}
+
+impl Copy for SchemaNameError {}
+
+/// Custom errors related to `SchemaDescription`.
+#[derive(Clone, Error, Debug)]
+pub enum SchemaDescriptionError {
+    /// Encountered a too long schema description.
+    #[error("Schema description contains more than 256 characters")]
+    TooLongSchemaDescription,
+}
+
+impl Copy for SchemaDescriptionError {}
```

Comment:
I'm not sure about this particular case, but I realized that the `Copy` trait is often not required and implemented because of copy & pasting ☺️

Think it's only required if the error types contain additional data?

Just saying, if you already checked and it's required then all good.

Reply:
I did it cos clippy told me to 😅 it's not needed, but it also wasn't a copy pasty error.... I guess we can switch that clippy flag off if we like.